redbot/cogs/mod/settings.py

```diff
@@ -365,3 +367,15 @@ async def defaultdays(self, ctx: commands.Context, days: int = 0):
                 days=days
             )
         )
+
+    @modset.command()
+    @commands.guild_only()
+    async def defaultduration(self, ctx: commands.Context, days: int = 0):
```

Comment:
I think it would be nicer to just use `TimedeltaConverter` here and save `total_seconds()`. I wouldn't be too surprised if someone wanted to use a few hours long duration as default and this seems like a rather arbitrary limitation to pose here.